phongo_compat.h

```diff
@@ -156,7 +156,10 @@
 # define SUPPRESS_UNUSED_WARNING(x)
 # define DECLARE_RETURN_VALUE_USED int return_value_used = 1;
 # define EXCEPTION_P(_ex, _zp) ZVAL_OBJ(_zp, _ex)
-# define PHONGO_STREAM_ID(stream) stream->res->handle
+//TODO - PHPC-401 - Revert this change when this issue is closed
+//# define PHONGO_STREAM_ID(stream) stream->res->handle
+# define PHONGO_STREAM_ID(stream) 42
+//End Workaround
```

Comment:
No need for the inline TODO, we have the ticket.
Besides, // is not a valid comment in c89 :)